src/components/Editor/Invitees/AvatarParticipationStatus.vue

```diff
@@ -143,14 +143,14 @@ export default {
 			if (this.isViewedByOrganizer) {
 				return {
 					class: ['no-response', 'icon', 'icon-invitees-no-response-white'],
-					label: t('calendar', '{name} hasn\'t responded yet.', {
+					label: t('calendar', 'Invitation sent', {
 						name: this.commonName,
 					}),
 				}
 			} else {
 				return {
 					class: ['no-response', 'icon', 'icon-invitees-no-response-white'],
-					label: t('calendar', '{name} did not respond to {organizerName}\'s invitation yet.', {
+					label: t('calendar', 'did not respond to {organizerName}\'s invitation yet.', {
```

Comment:
```suggestion
					label: t('calendar', 'Has not respond to {organizerName}\'s invitation yet.', {
```

Reply:
damn, you;re fast, i was about to write what do you think about the string that has the organizer's name and if its alright to keep it like this. But i guess you just approved :P